eZ/Publish/Core/IO/TolerantIOService.php

```diff
@@ -73,7 +73,7 @@ public function loadBinaryFile($binaryFileId)
     {
         $this->checkBinaryFileId($binaryFileId);
 
-        if ($binaryFileId[0] === '/') {
+        if ($binaryFileId[0] === '/' || (PHP_OS === 'WINNT' && $binaryFileId[1] === ':')) {
```

Comment:
you forgot to also use it in https://github.com/ezsystems/ezpublish-kernel/pull/1725/files#diff-25849153f4c0cf8e383a0523e8de9c24L159